sample_size/scripts/sample_size_run.py

```diff
@@ -14,20 +14,23 @@ def main() -> None:
     """
     from sample_size.sample_size_calculator import SampleSizeCalculator
     from sample_size.scripts.input_utils import get_alpha
-    from sample_size.scripts.input_utils import get_metric_metadata
+    from sample_size.scripts.input_utils import get_metrics
+    from sample_size.scripts.input_utils import get_variants
 
     try:
         # Get alpha for power analysis
         alpha = get_alpha()
-        calculator = SampleSizeCalculator(alpha)
+        variants = get_variants()
+        calculator = SampleSizeCalculator(alpha, variants)
```

Comment:
the second attribute should be power. You will need to update SampleSizeCalculator to take variants.